src/app/components/feed/jh-main-feed.directive.js

```diff
@@ -33,8 +35,44 @@
           Janus.attachMediaStream(video, newVal);
         }
       });
+      
+      $timeout(function(){
+        var video = $('video', element)[0];
+        var mainFeed = document.getElementById('main-feed');
+        video.addEventListener('dblclick', function() {
```

Comment:
Why not bind/catch the event directly to `main-feed`? Due to bubbling principle, when you'll make double click on `video` element it will be run also on its ancestors unless the propagation be stopped.